From 3bddf152b902d8a1e0f674b792577b2cca635c4b Mon Sep 17 00:00:00 2001
From: Brandon Bergren <xfce@bdragon.rtk0.net>
Date: Sun, 26 Aug 2018 10:28:05 -0400
Subject: [PATCH] fix build failure when dbus is not in the system include path
 (Bug #14386)

The d-bus direct depenencies were taken out in 91860af3a38080,
but the #include <dbus/dbus.h> in xfce4-session/xfsm-global.h
and the #include <dbus/dbus-glib.h> line in xfsm-error.c got left
in accidentally.
---
 xfce4-session/xfsm-error.c  | 2 --
 xfce4-session/xfsm-global.h | 1 -
 2 files changed, 3 deletions(-)

diff --git a/xfce4-session/xfsm-error.c b/xfce4-session/xfsm-error.c
index 2b7f6c04..f0766f5f 100644
--- a/xfce4-session/xfsm-error.c
+++ b/xfce4-session/xfsm-error.c
@@ -20,8 +20,6 @@
 #include <config.h>
 #endif
 
-#include <dbus/dbus-glib.h>
-
 #include <xfce4-session/xfsm-error.h>
 
 #define XFSM_DBUS_NAME "org.xfce.SessionManager"
diff --git a/xfce4-session/xfsm-global.h b/xfce4-session/xfsm-global.h
index d4587123..610d5bf7 100644
--- a/xfce4-session/xfsm-global.h
+++ b/xfce4-session/xfsm-global.h
@@ -27,7 +27,6 @@
 #include <X11/SM/SMlib.h>
 
 #include <xfce4-session/xfsm-splash-screen.h>
-#include <dbus/dbus.h>
 
 typedef struct _FailsafeClient FailsafeClient;
 struct _FailsafeClient
-- 
2.19.0

